src/server/jobs/prepare-bounties.ts

```diff
@@ -154,8 +298,22 @@ const prepareBounties = createJob('prepare-bounties', '0 1 * * *', async () => {
       }
     }
 
+    if (user) {
+      bountyAutomaticallyAwardedEmail.send({
+        bounty: {
+          id,
+          name,
+        },
+        entry: {
+          id: winnerEntryId,
+        },
+        user: {
+          email: user.email,
+        },
+      });
```

Comment:
Since we are not awaiting this, we could catch the error and log it somewhere in case we need to know the reason why
```suggestion
      bountyAutomaticallyAwardedEmail.send({
        bounty: {
          id,
          name,
        },
        entry: {
          id: winnerEntryId,
        },
        user: {
          email: user.email,
        },
      }).catch((error) => {
        // log error...
      })
```